### PR TITLE
Add llama model change

### PR DESCRIPTION
**Pull request by yeonsily**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

### CODE REVIEW

File: optimum/habana/transformers/generation/utils.py [hunk 1]
```diff
@@ -179,6 +180,35 @@ def _update_model_kwargs_for_generation(
 
         return model_kwargs
 
+    def _validate_model_kwargs(self, model_kwargs: Dict[str, Any]):
```
Comment (regisss) [inline optimum/habana/transformers/generation/utils.py:183]: The issue with `token_type_ids` returned by Llama's tokenizer has been solved in https://github.com/huggingface/transformers/pull/24042, which will be part of the next release of Transformers. So let's add the following comment to not forget it:
```suggestion
    # TODO: remove this method when Transformers v4.31 is released since it solves the issue with Llama
    def _validate_model_kwargs(self, model_kwargs: Dict[str, Any]):
```

**Review by regisss — COMMENTED** [review]
@yeonsily LGTM!

I just have a minor remark about adding a comment in the code that will have to be updated when Transformers v4.31 is released.

Also, I see that the code style check failed. Could you run
```bash
pip install black ruff
make style
```
from the root of the repo please?

File: optimum/habana/transformers/generation/utils.py (hunk 1 above)
Comment (yeonsily) [reply optimum/habana/transformers/generation/utils.py:183]: @regisss  Thank you for the update! Will add that.
Btw I saw 'make style' update other code format not just my changes in my files. Let me try again.

**Review by regisss — APPROVED** [review]
LGTM!
